### PR TITLE
Create bug_template.yml

### PR DESCRIPTION
**Pull request by PierrunoYT**
Can you test this template?

### CODE REVIEW

**Review by neubig — COMMENTED** [review]
To clarify, we already have this: https://github.com/OpenDevin/OpenDevin/blob/main/.github/ISSUE_TEMPLATE/bug_report.md

Why do we need another one?

**Comment by PierrunoYT** [comment]
> To clarify, we already have this: https://github.com/OpenDevin/OpenDevin/blob/main/.github/ISSUE_TEMPLATE/bug_report.md
> 
> 
> 
> Why do we need another one?

Linking this https://github.com/OpenDevin/OpenDevin/issues/981

**Comment by rbren** [comment]
@neubig this would turn our existing template into a _form_, which will be much more likely to be filled out 😄

**Comment by rbren** [comment]
@PierrunoYT you should also remove the existing template in this PR

File: .github/ISSUE_TEMPLATE/bug_template.yml [hunk 1]
```diff
@@ -0,0 +1,112 @@
+name: 🐞 Bug Report
+description: File a bug report to help us improve
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
```
Comment (rbren) [inline .github/ISSUE_TEMPLATE/bug_template.yml:10]:
```suggestion
```
Appreciate the sentiment here, but this will just clutter the issue a bit 😄

File: .github/ISSUE_TEMPLATE/bug_template.yml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        #### Describe the bug
+        <!-- a short description of the problem -->
+
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Bug Description
+      description: Provide a short description of the problem
+    validations:
+      required: true
```
Comment (rbren) [inline .github/ISSUE_TEMPLATE/bug_template.yml:20]: This seems like a dupe of the above?

**Comment by PierrunoYT** [comment]
> @PierrunoYT you should also remove the existing template in this PR

Okay

**Comment by PierrunoYT** [comment]
> @neubig this would turn our existing template into a _form_, which will be much more likely to be filled out 😄

Not sure if it's done correctly.

**Comment by rbren** [comment]
@PierrunoYT it seems like several sections are duplicated

**Comment by PierrunoYT** [comment]
> @PierrunoYT it seems like several sections are duplicated

It should be correct. We need to test how it would looks like and if it's wrong will check again.

**Comment by rbren** [comment]
For example, you have this:
```
  - type: markdown
    attributes:
      value: |
        #### Setup and configuration
        **Current version**:
        <!-- run `git log -n 1` to see this -->
  - type: code
    id: current-version
    attributes:
      label: Current Version
      description: Run `git log -n 1` to see this
      render: bash
```

IIUC this is going to create two different sections where we ask the user to run `git log -n 1`

**Comment by rbren** [comment]
There are several problems with the template listed here: https://github.com/OpenDevin/OpenDevin/blob/5d9c7463037965559cf81f8164480298d5aaec9f/.github/ISSUE_TEMPLATE/bug_template.yml

**Comment by PierrunoYT** [comment]
> There are several problems with the template listed here: https://github.com/OpenDevin/OpenDevin/blob/5d9c7463037965559cf81f8164480298d5aaec9f/.github/ISSUE_TEMPLATE/bug_template.yml

Tried to update it again but I do it all whith AI so it's might be wrong.

**Comment by rbren** [comment]
Apparently AI isn't getting this one right 😄 

If you click "view file" next to your bug_template.yml, you can test it out (currently it reports several errors).

But maybe we should just close this if the AI can't handle it

**Comment by PierrunoYT** [comment]
> Apparently AI isn't getting this one right 😄 
> 
> 
> 
> If you click "view file" next to your bug_template.yml, you can test it out (currently it reports several errors).
> 
> 
> 
> But maybe we should just close this if the AI can't handle it

Let me try again but maybe someone else has experience with yml and can fix it.

**Comment by PierrunoYT** [comment]
> Thanks @Umpire2018! This is super helpful.
> 
> Looks like we just have a merge conflict

Not sure how to fix it.
